test/UnitTest/Extensions/InstanceExtentions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json;
+using Altinn.Platform.Storage.Interface.Models;
+
+namespace Altinn.Platform.Storage.UnitTest.Extensions
+{
+    public static class InstanceExtentions
+    {
+        public static Instance Clone(this Instance instance)
```

Comment:
The clone will now have an identical ID. Probably fine in most cases, but there might still be some interference between tests that share an actual PostgreSQL server and database.

Reply:
The cloning is just because the instance id is updated before it's returned in the repository class, and if several tests are run in parallell, the test data gets corrupted.